src/utils/btoa.ts

```diff
@@ -0,0 +1,3 @@
+const btoa = (data: string): string => Buffer.from(data).toString('base64')
```

Comment:
This will fail in browser environment. I'm not sure if anybody uses this library in browser, but this would be a breaking change.

https://github.com/MunifTanjim/node-bitbucket/issues/84 mentions `@node-fetch/btoa-lite` which can be used as a drop-in replacement for `btoa-lite`

Reply:
Ok, @MunifTanjim I will just change packages